deploy/base/apps/istio/istio-ingress.yaml

```diff
@@ -23,16 +23,16 @@ spec:
       k8s:
         resources:
           limits:
-            cpu: 200m
+            cpu: PLACEHOLDER
```

Comment:
Can this PLACEHOLDER be something more descriptive? or was given this name for a reason?

Reply:
We agreed to use "PLACEHOLDER" on base files for now, any suggestions are welcome

Reply:
Cool, my question was more in regards of using the same PLACEHOLDER tag? But if it this was already reviewed and agreed with the team, I am good 👍🏻